pkg/app/pipectl/cmd/application/list.go

```diff
@@ -52,6 +53,7 @@ func newListCommand(root *command) *cobra.Command {
 	cmd.Flags().StringVar(&c.appName, "app-name", c.appName, "The application name.")
 	cmd.Flags().StringVar(&c.envID, "env-id", c.envID, "The environment ID.")
 	cmd.Flags().StringVar(&c.appKind, "app-kind", c.appKind, fmt.Sprintf("The kind of application. (%s)", strings.Join(model.ApplicationKindStrings(), "|")))
+	cmd.Flags().BoolVar(&c.disabled, "disabled", c.disabled, "Whether to show disabled applications or not.")
```

Comment:
nit: At first it looked to return both disabled and enabled to me. I guess it's better to emphasize like:

```suggestion
	cmd.Flags().BoolVar(&c.disabled, "disabled", c.disabled, "Show only disabled applications.")
```

But it's on you.

Reply:
Hmm, this one's a little more confusing as well. Never mind!

Reply:
Nice. Just updated.